LICENSE

```diff
@@ -0,0 +1,3 @@
+This work is licensed under the Creative Commons Attribution-ShareAlike 3.0 Unported License. 
+To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/3.0/ or 
```

Comment:
Wait! https!

Reply:
done. if you can confirm, I'll merge. thx!